content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -84,7 +100,7 @@ export class ContextMenuItem extends React.PureComponent {
     const {option} = this.props;
     return (
       <li role="menuitem" className="context-menu-item" >
-        <button className={option.disabled ? "disabled" : ""} onClick={this.onClick} onKeyDown={this.onKeyDown} tabIndex="0" >
+        <button className={option.disabled ? "disabled" : ""} tabIndex="0" onClick={this.onClick} onKeyDown={this.onKeyDown}>
```

Comment:
what's the reason for this change?

Reply:
I'm not sure, I may have removed the tabIndex to test something and then put it back :)